e2e/app/page/cohort-actions-page.ts

```diff
@@ -15,19 +15,16 @@ export default class CohortActionsPage extends AuthenticatedPage {
   }
 
   async isLoaded(): Promise<boolean> {
-    try {
-      await Promise.all([
-        waitForDocumentTitle(this.page, PageTitle),
-        waitWhileLoading(this.page),
-        this.getCreateAnotherCohortButton().then(elemt => elemt.asElementHandle()),
-        this.getCreateReviewSetsButton().then(elemt => elemt.asElementHandle()),
-        this.getCreateDatasetButton().then(elemt => elemt.asElementHandle()),
-      ]);
-      return true;
-    } catch (e) {
-      console.log(`CohortActionsPage isLoaded() encountered ${e}`);
-      return false;
-    }
+    await Promise.all([
```

Comment:
why was try and catch remove?

Reply:
Error caught in catch block is not handled and not passed on. Test continued and it made troubleshooting much harder to find what happened. So I want the test to stop when error happens and let `handleTestEvent` capture screenshot and save page content in a file.